platforms/ide/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/AbstractClientProvidedBuildActionRunner.java

```diff
@@ -68,26 +66,10 @@ protected interface ClientAction {
         boolean isRunTasks();
     }
 
-    private static class ActionResults {
```

Comment:
ℹ️  The class was only used to replay the dispatch of the phase results in case of a cache hit, and is not needed anymore, since this happens transparently.